### PR TITLE
Add parameter for skipping the tagRepository deployment

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Add parameter `skipTagRepository` for skipping TagRepository step. This with other parameters will allow for effectively skipping the entire release stage of the pipeline.

### CODE REVIEW

File: eng/pipelines/templates/stages/archetype-net-release.yml [hunk 1]
```diff
@@ -43,51 +43,52 @@ stages:
...
-                          -Artifact ${{artifact.name}}
-                      condition: and(succeeded(),ne('${{ artifact.skipPublishPackage }}', 'true'))
-                      displayName: Verify Package Installation
-                    - template: /eng/common/pipelines/templates/steps/create-tags-and-git-release.yml
-                      parameters:
-                        ArtifactLocation: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.name}}
-                        PackageRepository: Nuget
-                        ReleaseSha: $(Build.SourceVersion)
-                        RepoId: Azure/azure-sdk-for-net
+          - ${{if ne(artifact.skipTagRepository, 'true')}}:
```
Comment (benbp) [inline eng/pipelines/templates/stages/archetype-net-release.yml:46]: Should `Skip.Release` also affect this deployment or no?

Comment (benbp) [reply eng/pipelines/templates/stages/archetype-net-release.yml:46]: Also just for my own context, is the idea that we want to be able to permanently skip tagging for some artifacts, or that a user would update the ci.yml file in the pipeline page before running?

Comment (chidozieononiwu) [reply eng/pipelines/templates/stages/archetype-net-release.yml:46]: The idea is to be able to skip a release for a single artifact, permanently in this case.

**Review by benbp — APPROVED** [review]

**Comment by weshaggard** [comment]
@chidozieononiwu is this the only part that doesn't currently have a skip option?

**Comment by chidozieononiwu** [comment]
> @chidozieononiwu is this the only part that doesn't currently have a skip option?
yes. as far as the release stage is concerned.

**Pull request merged**